_sass/blocks/explore/_regions-list.scss

```diff
@@ -165,7 +185,58 @@
   }
 }
 
-.red {
-  color: $red;
+.bold {
   font-weight: $weight-bold;
 }
+
+.material-variance {
+  border-right: 2px solid $neutral-gray;
+}
+
+.threshold {
+  display: inline-block;
+  font-weight: $weight-bold;
+  margin-left: $base-padding-lite;
+}
+
+.revenue-type-list {
+  tr .variance {
+    text-align: right;
+    width: 40px;
+  }
+
+  tr:first-of-type {
+    border-top: 1px solid $neutral-gray;
+  }
+}
+
+.detached-header {
+  position: relative;
+}
+
+.detached-table_title {
+  float: left;
+  width: 50%;
+
+  @include respond-to(small-up) {
+    width: 60%;
+  }
+
+  @include respond-to(medium-up) {
+    width: 75%;
+  }
+
+  @include respond-to(large-up) {
+    width: 100%;
+  }
+}
+
+.detached-table_header {
+  display: inline;
```

Comment:
Properties should be ordered bottom, display, float, position, right, text-align, vertical-align